tests/test_pip_step_decorator.py

```diff
@@ -1,9 +1,9 @@
-"""Test pip step decorator."""
+"""Test pip step decorator as independent feature."""
```

Comment:
This file now only contains tests for `@pip` in isolation.
Flows using `@pip` and project environment are with the main tests in `test_environment.py`